src/org/dotnet/ci/triggers/GenericTriggerBuilder.groovy

```diff
@@ -7,7 +7,8 @@ import jobs.generation.Utilities
 // Periodic - Periodic triggers against a Git repo
 class GenericTriggerBuilder implements TriggerBuilder {
```

Comment:
Out of curiosity, do you see other types of Triggers being created in the future? Will it be better to refactor this into an interface and make Periodic and Manual extend from `GenericTriggerBuilder`? Is that even possible in Groovy? :|